pkg/deploy/lattice/target_group_manager.go

```diff
@@ -382,3 +411,44 @@ func (s *defaultTargetGroupManager) findTargetGroup(ctx context.Context, targetG
 	}
 	return nil, nil
 }
+
+// Get default health check configuration according to
+// https://docs.aws.amazon.com/vpc-lattice/latest/ug/target-group-health-checks.html#health-check-settings
+func (s *defaultTargetGroupManager) getDefaultHealthCheckConfig(targetGroupProtocolVersion string, targetGroupPort int64) *vpclattice.HealthCheckConfig {
+	var intResetValue int64 = 0
+
+	defaultMatcher := vpclattice.Matcher{
+		HttpCode: aws.String("200"),
+	}
+
+	defaultPath := "/"
+
+	defaultProtocol := vpclattice.TargetGroupProtocolHttp
+
+	isTgProtocolVersionEmptyOrGrpc := targetGroupProtocolVersion == "" || targetGroupProtocolVersion == vpclattice.TargetGroupProtocolVersionGrpc
+
+	if isTgProtocolVersionEmptyOrGrpc {
+		targetGroupProtocolVersion = vpclattice.HealthCheckProtocolVersionHttp1
+	}
+
+	enabled := targetGroupProtocolVersion == vpclattice.TargetGroupProtocolVersionHttp1
+
```

Comment:
Seems from current code logic, the Grpc defauit HealthCheckConfig Enabled==true? I think it should be false? 



Would be good if you can also a unit test for getDefaultHealthCheckConfig

Reply:
thanks for the catch, I'll fix this, and add unit tests